pages/404.tsx

```
@@ -16,6 +22,12 @@ const NotFound: NextPage = () => {
           <div className="text-center">
             <h1 className="title">404 - Page Not Found</h1>
             <p className="mt-2">The page you are looking for does not exist.</p>
+            <button
+              className="mt-4 px-4 py-2 bg-blue-500 text-white rounded hover:bg-blue-600"
+              onClick={() => (window.location.href = "/")}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)